tests/sharding/run.sh

```diff
@@ -80,17 +80,15 @@ function run() {
 
 	# resume-task to next stage
 	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"resume-task test" \
-		"\"result\": true" 3
+		"resume-task test"
 
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"query-status test" \
 		"failpoint error for FlushCheckpointStage before track DDL" 1
 
 	# resume-task to next stage
 	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"resume-task test" \
-		"\"result\": true" 3
```

Comment:
why don't you need to test the number of `result: true`
Is it because the following is tested with `dmctl query-status`?

Reply:
see PR description